### PR TITLE
Display Warning when a Bulk has Expired

### PR DESCRIPTION
**Pull request by Dala0**
Display a warning icon when a bulk was restored more than the number of days set in the server configuration as the expiry time in the past. When the user moves their mouse over the icon, pop-up text is displayed which explains why the icon is there.

Includes a new endpoint for returning information about the server configuration. It currently only contains the expiry time in days.

image

Tested on a machine running macOS 10.12.6.

@fredex42

### CODE REVIEW

**Comment by Dala0** [comment]
[Add warning to bulk selector tab if it was added longer ago than the glacier keep-online time](https://trello.com/c/XvE3mpK3/3099-add-warning-to-bulk-selector-tab-if-it-was-added-longer-ago-than-the-glacier-keep-online-time)

File: app/controllers/ConfigController.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                                 override val wsClient:WSClient)
+  extends AbstractController(controllerComponents) with Circe with PanDomainAuthActions {
+
+  /**
+    * Endpoint that returns configuration settings
+    * @return
+    */
+  def getConfig = APIAuthAction { request=>
+    val defaultExpiry = config.getOptional[Int]("archive.restoresExpireAfter").getOrElse(3)
+    val configSettings = List(defaultExpiry.toString)
```
Comment (fredex42) [inline app/controllers/ConfigController.scala]: fair enough for just one setting but I think it's better to use a JSON object e.g. Map("defaultExpiry"->defaultExpiry). Also no need to convert to a string if its JSON.  Not worth changing right now though

File: frontend/app/Lightbox/BulkSelectionsScroll.jsx [hunk 2]
```diff
@@ -95,6 +108,7 @@ class BulkSelectionsScroll extends React.Component {
                             evt.stopPropagation();
                             this.props.onDeleteClicked(entry.id);
                         }}/>
+                        <span data-tip="Some or all of the media may have returned to the deep archive and may be unavailable."><ReactTooltip/><FontAwesomeIcon icon="exclamation-triangle" style={{marginRight: "6px" ,color: "red", float: "left", display: this.showExpiryWarning(entry.addedAt)}} /></span>
```
Comment (fredex42) [inline frontend/app/Lightbox/BulkSelectionsScroll.jsx]: tiny request for changing wording - Some or all of the media may have returned to the deep archive and *therefore* be unavailable.

Comment (Dala0) [reply frontend/app/Lightbox/BulkSelectionsScroll.jsx]: Done.

**Pull request merged**